Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
 resolver = "2"
-members = [
-  "p2panda-core",
-]
+members = ["p2panda-core", "p2panda-core/fuzz"]
```

Comment:
Should we move the `fuzz` folder into the root of this repository (potentially holding more fuzz tests for other crates?)